src/software/jetson_nano/rotary_encoder/RotaryEncoder.py

```diff
@@ -0,0 +1,141 @@
+import Jetson.GPIO as GPIO
+
+"""
+The state of a rotary encoder is dependent on the values read on pin A and pin B.
+The input read on pin A and pin B can either be 0 or 1 and therefore we have four
```

Comment:
Is there a link to a datasheet to explain this pin A pin B naming?

Reply:
fixed